ui/app/styles/components/dashboard-metric.scss

```
@@ -17,6 +17,12 @@
     font-weight: $weight-bold;
     font-size: $size-3;
 
+    &.topo {
```

Nit-picking on the name here a bit, the `dashboard-metric` class is already restricted to the topoviz page, so `topo` doesn't describe this class very well. It also leaks some assumption on where this is used.

Looking at the page, what differentiates this `metric` from other (like CPU and memory) is that they are justified so maybe naming it so would make the intention more clear?

```suggestion
    &.justify {
```

---

Fixed in https://github.com/hashicorp/nomad/pull/17634